sampleproject/components/calendar/calendar.py

```diff
@@ -25,6 +25,7 @@ def get(self, request, *args, **kwargs):
                 kwargs={
                     "date": request.GET.get("date", ""),
                 },
+                type="append",
```

Comment:
WIth the new `"prepend"` and `"append"` options, it's possible to render a component ad hoc, without using neither the `{% component_dependencies %}` placeholders, nor requiring `<head>` and `<body>` HTML tags.

These types simply prepend or append all associated JS / CSS.